### PR TITLE
Handle repos changes such as name renaming issue #6 

### PR DESCRIPTION
**Pull request by samueldaviddelacruz**
also fixed a typo on the readme :)

### CODE REVIEW

File: src/server/models/index.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ module.exports = {
     timestamps: false
   }),
   Repo: sequelize.define('repo', {
-    id: { type: Sequelize.INTEGER, primaryKey: true },
+
+    id: { type: Sequelize.STRING(200), primaryKey: true },
```
Comment (rmariuzzo) [inline src/server/models/index.js]: Hey @samueldaviddelacruz! Thanks for contributing to this project, that's awesome! Can you remove the extra empty lines added here. We just want to be consistent with the code style.

File: src/server/services/github/queries/index.js [hunk 2]
```diff
@@ -52,12 +52,14 @@ module.exports = {
   /**
    * Graphql query to search repositories on GitHub API.
    */
+
```
Comment (rmariuzzo) [inline src/server/services/github/queries/index.js]: The same here, can we remove this empty line?

File: src/server/services/repo/index.js [hunk 3]
```diff
@@ -11,22 +11,23 @@ module.exports = {
    * @return {Promise} A promise.
    */
   createRepos(repos = []) {
-    print.trace(`Create ${repos.length} repos...`)
+    print.trace(`Create ${repos.length} repos...`);
```
Comment (rmariuzzo) [inline src/server/services/repo/index.js]: We are not using `;` can you remove it? Please.

File: src/server/services/repo/index.js [hunk 4]
```diff
@@ -11,22 +11,23 @@ module.exports = {
...
       // Prevent creating repo with same name.
-      const existingNames = results.map(r => r.name)
-      const newRepos = repos.filter(r => existingNames.indexOf(r.name) === -1)
+      const existingIds = results.map(r => r.id);
+      const newRepos = repos.filter(r => existingIds.indexOf(r.id) === -1);
 
       return Repo.bulkCreate(
-        newRepos.map((repo) => ({
+          newRepos.map((repo) => ({
+            id:repo.id,
```
Comment (rmariuzzo) [inline src/server/services/repo/index.js]: Check the indentation here, please.

File: src/server/services/repo/index.js [hunk 5]
```diff
@@ -11,22 +11,23 @@ module.exports = {
...
-        name: { $in: repos.map(r => r.name).filter(r => r) }
+        id: { $in: repos.map(r => r.id).filter(r => r) }
       }
     }).then((results) => {
 
       // Prevent creating repo with same name.
-      const existingNames = results.map(r => r.name)
-      const newRepos = repos.filter(r => existingNames.indexOf(r.name) === -1)
+      const existingIds = results.map(r => r.id);
+      const newRepos = repos.filter(r => existingIds.indexOf(r.id) === -1);
```
Comment (rmariuzzo) [inline src/server/services/repo/index.js]: The `;` 😛

File: src/server/models/index.js (hunk 1 above)
Comment (samueldaviddelacruz) [inline src/server/models/index.js]: fixed

File: src/server/services/github/queries/index.js (hunk 2 above)
Comment (samueldaviddelacruz) [inline src/server/services/github/queries/index.js]: fixed

File: src/server/services/repo/index.js (hunk 3 above)
Comment (samueldaviddelacruz) [inline src/server/services/repo/index.js]: fixed

File: src/server/services/repo/index.js (hunk 5 above)
Comment (samueldaviddelacruz) [inline src/server/services/repo/index.js]: fixed

File: src/server/services/repo/index.js (hunk 4 above)
Comment (samueldaviddelacruz) [inline src/server/services/repo/index.js]: fixed

**Comment by samueldaviddelacruz** [comment]
sorry for the ignorance and everything, do i need to make another pull request  after the changes?

**Comment by rmariuzzo** [comment]
@samueldaviddelacruz no need to create another PR, just push to your same branch

**Comment by samueldaviddelacruz** [comment]
OKs, i did.

**Comment by DannyFeliz** [comment]
PING @rmariuzzo :eyes:

**Pull request merged**